safe/Makefile

```diff
@@ -28,7 +28,9 @@ push:
 	docker push ${REPO}:${VERSION}
 
 deploy:
-	kubectl apply -f ./k8s/
+	kubectl apply -f ./k8s/Namespace.yaml
```

Comment:
Split this into 3 lines because ns needs to be created first; and I was too lazy to set up a Kustomize file.